src/Options.js

```diff
@@ -290,11 +290,16 @@ class Options {
      * Gets the CSS color of the current options object, accounting for the
      * `colorMap`.
      */
-    getColor(): string {
+    getColor(): ?string {
```

Comment:
I think this can still be `getColor(): string`.

Reply:
Not if `Options.color` is nullable…